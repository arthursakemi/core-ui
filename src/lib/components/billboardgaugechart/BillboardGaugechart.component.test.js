import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/billboardgaugechart/__snapshots__"
});
