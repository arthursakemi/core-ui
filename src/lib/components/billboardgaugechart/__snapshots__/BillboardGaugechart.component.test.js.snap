// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots billboard-gaugechart Default 1`] = `
<div>
  <h3>
    billboard gaugechart demo
  </h3>
  <div
    className="sc-bdVaJa bDWFJH sc-billboardgaugechart"
    id="gaugeChart"
  />
</div>
`;
